crux-metrics/src/crux/metrics/dropwizard/cloudwatch.clj

```diff
@@ -0,0 +1,36 @@
+(ns crux.metrics.dropwizard.cloudwatch
+  (:require crux.metrics.dropwizard)
+  (:import [io.github.azagniotov.metrics.reporter.cloudwatch CloudWatchReporter]
+           [java.io Closeable]
+           [software.amazon.awssdk.services.cloudwatch CloudWatchAsyncClient]
+           [software.amazon.awssdk.regions.providers DefaultAwsRegionProviderChain]
+           [software.amazon.awssdk.auth.credentials DefaultCredentialsProvider]
+           [java.util.concurrent TimeUnit]
+           [com.codahale.metrics MetricRegistry MetricFilter]))
+
+(defn ^CloudWatchReporter report
+  [^MetricRegistry reg {::keys [region dry-run? jvm-metrics?] :as args}]
+  (let [cwac (-> (CloudWatchAsyncClient/builder)
+                 (.region (.getRegion (new DefaultAwsRegionProviderChain)))
+                 (.credentialsProvider (DefaultCredentialsProvider/create))
+                 .build)]
+    ;; Might need seperate cwacs for regions
```

Comment:
you would, but I doubt we'll be reporting to more than one region on any one node